chart/templates/configmaps/extra-configmaps.yaml

```diff
@@ -37,13 +37,15 @@ metadata:
     {{- if $configMapContent.labels }}
       {{- toYaml $configMapContent.labels | nindent 4 }}
     {{- end }}
-  annotations:
-    "helm.sh/hook": "pre-install,pre-upgrade"
-    "helm.sh/hook-delete-policy": "before-hook-creation"
-    "helm.sh/hook-weight": "0"
-    {{- if $configMapContent.annotations }}
-      {{- toYaml $configMapContent.annotations | nindent 4 }}
-    {{- end }}
+  {{- $annotations := dict }}
+  {{- if $configMapContent.useHelmHooks }}
```

Comment:
If we do this, we must do it in such a way that we fall back to true if not specified for backcompat. And have test coverage for it.